File: TaskManager/LabFinal/LabFinal/CommandLog.cs
namespace LabFinal;

public class CommandLog
{
    public CommandType Type { get; set;}
    public string Description { get; set; }
    public DateTime Timestamp { get; set; }   
    
    public CommandLog(CommandType type, string description)
    {
        Type = type;
        Description = description;
        Timestamp = DateTime.Now;
    }
    
}

File: TaskManager/src/command/CommandLog.java
package command;

public class CommandLog {
    private final String description;
    private final String timestamp;
    private final CommandType type;



    public CommandLog(String description, String timestamp, CommandType type) {
        this.description = description;
        this.timestamp = timestamp;
        this.type = type;
    }

    public String getDescription() {
        return description;
    }

    public String getTimestamp() {
        return timestamp;
    }



    public CommandType getType() {
        return type;
    }

    @Override
    public String toString() {
        return "[" + timestamp + "] " + description + " (" + type + ")";
    }
}